README.md

```diff
@@ -6,6 +6,9 @@ See the original README-original.md for info about Godot.
 ## Releases
 See [the GitHub releases page](https://github.com/Stary2001/godot/releases) for pre-built editors for Windows, macOS and Linux.
 
+## What do I need in order to test my game on a real device?
+In order to playtest your own game on a physical Switch, you need to install a custom firmware on your Switch. This will most likely only work with very old models. More recent models need to have a modchip installed which is not very beginner friendly and also comes with legal problems. Therefore, we do not encourage any of these actions.
```

Comment:
This is just plainly incorrect.

Using any homebrew requires that you own a modded Switch. Every model of Switch can be modified for custom firmware. Newer models just require a mod chip because they are not susceptible to soft-modding, and that does *not* in fact come with any legal problems for the user.

Nintendo has gone after the ***manufacturers*** of mod chips in the past, but this implies nothing for the end-user.

In any case, this kind of disclaimer does not belong in the ReadMe of this project. Only a link to the [Switch hacking guide](https://nh-server.github.io/switch-guide) is necessary.